From ad6b34eea0710fefb584c44ac7e6a43501f14fb6 Mon Sep 17 00:00:00 2001
From: Jordan Petridis <jordan@centricular.com>
Date: Thu, 16 Dec 2021 22:37:42 +0200
Subject: [PATCH 1/2] docs: use gnomebt_api_name for the gtk-doc module name

Currently the docs are not prefixed with a version and overlap
with the gnome-bluetooth-1.0 files. gnomebt_api_name is like the
project name but with the api version appended and now the path looks
like:

`share/gtk-doc/html/gnome-bluetooth-2.0/index.html`
---
 docs/reference/libgnome-bluetooth/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/docs/reference/libgnome-bluetooth/meson.build b/docs/reference/libgnome-bluetooth/meson.build
index 51ad8089..94ce693a 100644
--- a/docs/reference/libgnome-bluetooth/meson.build
+++ b/docs/reference/libgnome-bluetooth/meson.build
@@ -20,7 +20,7 @@ version_xml = configure_file(
 )
 
 gnome.gtkdoc(
-  meson.project_name(),
+  gnomebt_api_name,
   main_sgml: meson.project_name() + '-docs.sgml',
   src_dir: lib_inc,
   dependencies: libgnome_bluetooth_dep,
-- 
2.31.1

